site/src/modules/resources/SSHButton/SSHButton.tsx

```diff
@@ -92,6 +82,15 @@ export const SSHButton: FC<SSHButtonProps> = ({
 	);
 };
 
+const SSHStep = (props: { helpText: string; codeExample: string }) => (
+	<li style={{ listStylePosition: "inside" }}>
+		<HelpTooltipText style={{ display: "inline" }}>
+			<strong css={styles.codeExampleLabel}>Connect to the agent:</strong>
```

Comment:
I think you meant to use the `helpText` prop here

Reply:
Oh jeeze, you're right 🤦‍♂️ 

Fixed in abf9abd58cf4ca7fdb54acebb5f5c45087ca4ab7